pkgs/applications/terminal-emulators/mlterm/default.nix

```diff
@@ -1,14 +1,82 @@
-{ stdenv, lib, fetchFromGitHub, pkg-config, autoconf, makeDesktopItem, nixosTests
-, libX11, gdk-pixbuf, cairo, libXft, gtk3, vte
-, harfbuzz #substituting glyphs with opentype fonts
-, fribidi, m17n_lib #bidi and encoding
+{ stdenv
+, lib
+, fetchFromGitHub
+, pkg-config
+, autoconf
+, makeDesktopItem
+, nixosTests
+, vte
+# Open Type layout support, (substituting glyphs with opentype fonts)
+, enableOtl ? true
+, harfbuzz # can be replaced with libotf
+, enableBidi ? true
+, fribidi
+, enableM17n ? !stdenv.isDarwin
+, m17n_lib
+, enableSsh2 ? true
```

Comment:
Nit: I prefer `withSsh2` but there's no standard here.

Reply:
Usually we follow the configuration script provided by the upstream software itself.
If they use `--enable-ssh2` we use `enableSsh2`.
If they use `--with-hugetlb` we use `withHugetlb`.

Reply:
> Usually we follow the configuration script provided by the upstream software itself. If they use `--enable-ssh2` we use `enableSsh2`. If they use `--with-hugetlb` we use `withHugetlb`.

I agree, though the `--with-x=yes` is an exception in this particular package.